src/Features/Core/Portable/Diagnostics/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -25,24 +25,24 @@ internal static class IDEDiagnosticIds
         public const string InlineAsTypeCheckId = "IDE0019";
         public const string InlineIsTypeCheckId = "IDE0020";
 
-        public const string UseExpressionBodyForConstructorsDiagnosticId = "IDE0020";
-        public const string UseExpressionBodyForMethodsDiagnosticId = "IDE0021";
-        public const string UseExpressionBodyForConversionOperatorsDiagnosticId = "IDE0022";
-        public const string UseExpressionBodyForOperatorsDiagnosticId = "IDE0023";
-        public const string UseExpressionBodyForPropertiesDiagnosticId = "IDE0024";
-        public const string UseExpressionBodyForIndexersDiagnosticId = "IDE0025";
-        public const string UseExpressionBodyForAccessorsDiagnosticId = "IDE0026";
+        public const string UseExpressionBodyForConstructorsDiagnosticId = "IDE0021";
```

Comment:
Can we avoid changing all the IDs and instead just bump the duplicate ID to a new one? Otherwise, I am worried this will break existing suppressions for all these IDs.